pymeasure/instruments/hp/hp856Xx.py

```diff
@@ -0,0 +1,304 @@
+        maxsplit=0
+    )
+
+    attenuation = Instrument.control(
+        "AT?", "AT %s",
+        """
+        Controls input attenuation in decade steps from 10 to 70db or set to AUTO and MAN(ual)
+        as well as incremental control via UP/DN
+        """,
+        validator=joined_validators(strict_discrete_set, truncated_discrete_set),
+        values=[["AUTO", "MAN", "UP", "DN"], [10, 20, 30, 40, 50, 60, 70]]
+    )
+
+    amplitude_unit = Instrument.control(
+        "AUNITS?", "AUNITS %s",
+        """
+        Controls the amplitude unit with a selection of the following parameters:
+        'DBM', 'DBMV', 'DBUV', 'V', 'W', 'AUTO', 'MAN'
+        and affects the following properties:
+        """,
+        validator=strict_discrete_set,
+        set_process=lambda v: v.value,
+        values=[e for e in AmplitudeUnits]
+    )
+
+    def auto_couple(self):
+        """
+        Sets video bandwidth, resolution bandwidth, input attenuation,
+        sweep time, and center frequency step-size to coupled mode. These functions can be recoupled
+        individually or all at once. The spectrum analyzer chooses appropriate values for these
+        functions. The video bandwidth and resolution bandwidth are set according to the coupled
+        ratios stored under TODO and TODO.
+        If no ratios are chosen, default ratios (1.0 and 0.011, respectively) are used instead.
+        """
+        self.write("AUTOCPL")
+
+    def exchange_traces(self):
+        """
+        Exchanges the contents of trace A with those of trace B. If the traces are
+        in clear-write or max-hold mode, the mode is changed to view. Otherwise, the traces remain
+        in their initial mode.
+        """
+        self.write("AXB")
+
+    def blank_trace(self, trace: Trace):
+        """
+        Blanks the chosen trace from the display. The current contents of the
+        trace remain in the trace but are not updated.
+        trace: Takes type 'Trace' selecting the trace
+        """
+        self.write("BLANK " + trace)
+
+    def subtract_display_line_from_trace_b(self):
+        """
+        Subtracts the display line from trace B and places the result in dBm
+        (when in log mode) in trace B, which is then set to view mode. In linear mode, the results
+        are in volts.
+        """
+        self.write("BML")
+
+    freq_center = Instrument.control(
+        "CF?", "CF %s",
+        """
+        Sets the center frequency in hertz and sets the spectrum analyzer to center
+        frequency / span mode. The span remains constant; the start and stop frequencies change as
+        the center frequency changes.
+        """,
+        validator=joined_validators(strict_range, strict_discrete_set),
+        values=[[0, 1], ["UP", "DN"]],
+        dynamic=True,
+        set_process=lambda v: v if isinstance(v, str) else (
+                '%.11E HZ' % Decimal(v)).replace('+', ''
+                                                 )
+    )
```

Comment:
1. Would'nt it sound better to call this property `center_frequency`
2. Spelling of the docstring (`Set` instead of `sets`, **H**ertz)
3. While Ilike the idea of allwoingthe limits also by "UP" & "DN", the whole `set_process` could be dropped if only numerical values are accepted. then the set-string could be written as `"CF %11.0f HZ"`

Reply:
Thanks for the feedback !
Regarding 1) I named the parameter the same as the Rohde & Schwarz FSL implementation, although I agree with your argument.
About 3) I think you are right, I will remove the relative modification parameters.
Would make more sense from an scientific view.